.azure/pipelines/quarantined-pr.yml

```diff
@@ -84,6 +85,7 @@ jobs:
     agentOs: Windows
     timeoutInMinutes: 90
     isTestingJob: true
+    enablePublishTestResults: false
```

Comment:
Similar to .azure/pipelines/components-e2e-tests.yml, the three non-Helix jobs in this pipeline need the extra `isTestingJob` steps but not the "normal" test publication (they handle publishing quarantined test results in this YAML). Change avoids similar warnings to those in .azure/pipelines/components-e2e-tests.yml.

Reply:
Technically, `isTestingJob`
1. turns on Arcade and JS test publication (unless overridden by `enablePublishTestResults`)
2. changes the Windows image from `Build.Server.Amd64.VS2019.BT.Open` to `Build.Server.Amd64.VS2019.Open`
3. runs InstallProcDump.ps1, StartDumpCollectionForHangingBuilds.ps1, and FinishDumpCollectionForHangingBuilds.ps1 on Windows build agents
4. runs InstallGoogleChrome.ps1 on Windows build agents

I could remove `isTestingJob: true` from all non-Windows jobs where `enablePublishTestResults: false`, leaving only a few `enablePublishTestResults: false` cases. Left this as-is because we may change how testing jobs are handled for Linux or macOS in the future and the split is pretty clean as I left it between (a) build jobs, (b) Helix submission jobs, and (c) "true" testing jobs (where `isTestingJob: true`).